examples/experimental/runtimeconfig/hello/helloBPWriterXML.cpp

```diff
@@ -17,22 +17,46 @@
 
 #include <adios2.h>
 
+#define str_helper(X) #X
+#define str(X) str_helper(X)
+
+#ifndef DEFAULT_CONFIG
+#define DEFAULT_CONFIG helloBPWriter.xml
+#endif
+#define DEFAULT_CONFIG_STR str(DEFAULT_CONFIG)
+
 int main(int argc, char *argv[])
 {
     MPI_Init(&argc, &argv);
     int rank, size;
     MPI_Comm_rank(MPI_COMM_WORLD, &rank);
     MPI_Comm_size(MPI_COMM_WORLD, &size);
 
+    std::string configFile;
+    if (argc == 1)
+    {
+        configFile = DEFAULT_CONFIG_STR;
+    }
+    else if (argc == 2)
+    {
+        configFile = argv[1];
+    }
+    else
+    {
+        std::cerr << "Usage: " << argv[0] << " [/path/to/config.xml]"
```

Comment:
if( rank == 0 )
{
    std::cerr << "Usage: " << argv[0] << " [/path/to/config.xml]"
                                         << std::endl;
}